book/reference/glossary.md

```diff
@@ -61,12 +61,19 @@
 
 ## Topical (Use Case Specific)
 ```{glossary}
-[basal slip](https://jupyterlab.readthedocs.io)
-  JupyterLab is the next-generation web-based user interface for Project Jupyter
-  intended to replace the JupyterNotebook interface.
-
-[subglacial water](https://jupyterlab.readthedocs.io)
-  JupyterLab is the next-generation web-based user interface for Project Jupyter
-  intended to replace the JupyterNotebook interface.
+[basal slip](https://en.wikipedia.org/wiki/Basal_sliding)
+  Basal slip or sliding is the act of a glacier sliding over the bed due to meltwater 
+  under the ice acting as a lubricant. This movement very much depends on the 
+  temperature of the area, the slope of the glacier, the bed roughness, the amount of 
+  meltwater from the glacier, and the glacier's size. 
+
+[subglacial water/lakes](https://en.wikipedia.org/wiki/Subglacial_lake)
+ A subglacial lake is a lake that is found under a glacier, typically beneath an ice
+ cap or ice sheet. Subglacial lakes form at the boundary between ice and the 
+ underlying bedrock, where gravitational pressure decreases the pressure melting
```

Comment:
There's a bit of debate on whether water from subglacial lakes are always at the ice-bed interface, or whether they're situated in porous sediment (a more aquifer-like setting).

Reply:
I don't think there's any debate now with [observational data of this phenomenon](https://doi.org/10.1126/science.abm3301). Feel free to amend this to the definition. It's worth mentioning!